stable/chartmuseum/templates/ingress.yaml

```diff
@@ -3,7 +3,7 @@
 {{- $serviceName := include "chartmuseum.fullname" . -}}
 {{- $ingressExtraPaths := .Values.ingress.extraPaths -}}
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
I think this is not mandatory for now. As kubernetes has switched to `networking.k8s.io/v1beta1` only in `1.14`. But most people run clusters of version bellow `1.14`

Reply:
I think that's fair, I will change back the API version for the **Ingress** resource since it will not be deprecated till `v.1.20`.